rocks-db/src/column.rs

```diff
@@ -512,4 +512,13 @@ where
         let res = db.get_cf(cf, &encoded_key)?;
         Ok(res.is_some())
     }
+
+    pub async fn key_exist(&self, key: C::KeyType) -> Result<bool> {
+        let db = self.backend.clone();
+        tokio::task::spawn_blocking(move || {
+            let cf = &db.cf_handle(C::NAME).unwrap();
+            let encoded_key = C::encode_key(key);
+            Ok(db.key_may_exist_cf(cf, &encoded_key))
```

Comment:
This may produce false positives. We already have the method 
```
  pub(crate) fn sync_has_key(db: Arc<DB>, key: C::KeyType) -> crate::Result<bool> {
        let cf = &db.cf_handle(C::NAME).unwrap();
        let encoded_key = C::encode_key(key);
        if !db.key_may_exist_cf(cf, &encoded_key) {
            return Ok(false);
        }
        let res = db.get_cf(cf, &encoded_key)?;
        Ok(res.is_some())
    }
```
I'd suggest wrapping it with async instead